x-pack/plugins/secops/public/components/timeline/body/renderers/total_virus_link.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexItem, EuiLink } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+const LinkFlexItem = styled(EuiFlexItem)`
+  margin-left: 6px;
+`;
+
+export const TotalVirusLink = pure(({ sha, value }: { sha: string; value: string }) => (
+  <LinkFlexItem grow={false}>
+    <EuiLink href={`https://www.virustotal.com/#/search/${encodeURI(sha)}`} target="_blank">
+      {value}
+    </EuiLink>
+  </LinkFlexItem>
+));
```

Comment:
Do you think `TotalVirusLink` and `GoogleLink` will have much re-use outside these renderers? If so, might be worthwhile to re-locate them to [components/links](https://github.com/elastic/kibana/tree/feature-secops/x-pack/plugins/secops/public/components/links). This is an optional as it can be refactored once those components start being used elsewhere, but just wanted to mention it.

Reply:
That looks like a good area to move them to. The renderers are getting too large with things that probably don't belong in there and that would encourage re-use.

Reply:
Moved it over to that folder